administrator/language/de-DE/com_cpanel.ini

```diff
@@ -5,38 +5,37 @@
 ; Note : All ini files need to be saved as UTF-8
 
 COM_CPANEL="Kontrollzentrum"
-COM_CPANEL_HEADER_SUBMENU="Untermenü"
-COM_CPANEL_HEADER_SYSTEM="System"
-COM_CPANEL_LINK_CHECKIN="Globales Freigeben"
-COM_CPANEL_LINK_CLEAR_CACHE="Cache leeren"
-COM_CPANEL_LINK_DASHBOARD="Kontrollzentrum"
-COM_CPANEL_LINK_EXTENSIONS="Erweiterungen installieren"
-COM_CPANEL_LINK_GLOBAL_CONFIG="Konfiguration"
-COM_CPANEL_LINK_SYSINFO="Systeminformationen"
-COM_CPANEL_MESSAGES_BODY_NOCLOSE="Es gibt wichtige Nachinstallationshinweise die einer Aufmerksamkeit bedürfen. Zum Anzeigen der Hinweise bitte auf „Hinweise anzeigen“ klicken."
+COM_CPANEL_ADD_DASHBOARD_MODULE="Module zum Kontrollzentrum hinzufügen"
```

Comment:
from here to line 18 i'm not sure it represents the correct meaning. lets see how it looks in beta.